2019-04-25-coopdevs-chat.md

```diff
@@ -0,0 +1,132 @@
+---
+source_url: https://hackmd.io/V9joJrlrSvWed0-ez2Phag
+---
+# 2019-04-25 Toronto Workers Co-op: CoopDevs Chat
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
+
+Time: Thu, Apr 25 @ 10-11am ET  
+Location: https://meet.jit.si/offline  
+Attending: patcon, dcwalk, ben&ferran, pau
+
+[TOC]
+
+## Agenda
+
+- Intros
+    - Hypha Workers Co-op
```

Comment:
```suggestion
    - Hypha Worker Co-op
```

Reply:
gah... i've definitely been one of the ppl sowing confusion with using the wrong word (had a misunderstanding earlier that's hard to break!)